docs/usage/shoot_settings/shoot_autoscaling.md

```diff
@@ -23,7 +28,7 @@ There are [general options for `cluster-autoscaler`](../api-reference/core.md#co
 ## Vertical Pod Auto-Scaling
 
 This form of auto-scaling is not enabled by default and must be explicitly enabled in the `Shoot` by setting `.spec.kubernetes.verticalPodAutoscaler.enabled=true`.
-The reason is that it was only introduced lately, and some end-users might have already deployed their own VPA into their clusters, i.e., enabling it by default would interfere with such custom deployments and lead to issues, eventually.
+The reason is that it was only introduced lately, and some end-users might have already deployed their own VPA into their clusters, i.e., enabling it by default would interfere with such custom deployments and lead to issues, eventually. Also if `ShootVPAEnabledByDefault` admissionPlugins is enabled then `.spec.kubernetes.verticalPodAutoscaler.enabled` will be set to true.
```

Comment:
```suggestion
The reason is that it was only introduced lately, and some end-users might have already deployed their own VPA into their clusters, i.e., enabling it by default would interfere with such custom deployments and lead to issues, eventually. Additionally, if the `ShootVPAEnabledByDefault` admission plugin is enabled in the gardener-apiserver, then the `.spec.kubernetes.verticalPodAutoscaler.enabled` field will be defaulted to true.
```

Add also a link to the admission plugin docs in `apiserver-admission-plugins.md`.